### ISSUE TITLE
High-Contrast Mode Doesn't Affect Brave

### ISSUE THREAD

**Issue by jonathansampson** [brave/browser-laptop#4302]
**Did you search for similar issues before submitting this one?**

Yes

**Describe the issue you encountered:**

Enabling any of the high-contrast profiles on Windows 10 leave Brave unaffected.

**Expected behavior:**

Enabling any of the high-contrast profiles on Windows 10 impacts Brave's UI.
- Platform (Win7, 8, 10? macOS? Linux distro?): Windows 10
- Brave Version: 0.12.2
- Steps to reproduce:
  1. Press Windows Key
  2. Type `Contrast`, Select `High contrast settings`
  3. Choose `High Contrast #1` theme
  4. Note that Brave is unimpacted
- Screenshot if needed:

![high-contrast](https://cloud.githubusercontent.com/assets/815158/18836739/263f5438-83b6-11e6-9c94-f2b3af466193.gif)
- Any related issues: #1754

**Comment by bsclifton** [comment]
Some quick thoughts while it's fresh:
- We should be able to detect the OS settings via Electron/Muon
- We could then expose these via the BrowserWindow object. Maybe an event like "onEnteringHighContrastMode".  We could also expose the object/color mapping via properties on the BrowserWindow object.
- browser-laptop would then respond to this; we would want to respect the values specified by the user

**Comment by bsclifton** [comment]
This works surprisingly well in [brave-core](https://github.com/brave/brave-browser)! 😄 
![high-contrast](https://user-images.githubusercontent.com/4733304/44942907-e84b6e00-ad70-11e8-85e7-ce51bc960e5f.gif)
